Documentation/devicetree/bindings/vendor-prefixes.yaml

```
@@ -618,6 +618,8 @@ patternProperties:
   "^grmn,.*":
     description: Garmin Limited
   "^gumstix,.*":
+    description: Guangdong Genius Technology Co.,Ltd.I
+  "^eebbk,.*":
     description: Gumstix, Inc.
```

`"^gumstix,.*":` should be together with `description: Gumstix, Inc.`

```suggestion
  "^eebbk,.*":
    description: Guangdong Genius Technology Co.,Ltd.
  "^gumstix,.*":
    description: Gumstix, Inc.
```